.travis.yml

```diff
@@ -50,8 +50,10 @@ script:
         echo "Don't build on coverty_scan branch.";
         exit 0;
     fi
-  - ./autogen.sh --prefix=/opt/yami --enable-h264enc --enable-jpegenc --enable-vp8enc --enable-vp9enc --enable-h265enc --enable-mpeg2dec --enable-vp8dec --enable-vp9dec --enable-jpegdec --enable-h264dec --enable-h265dec --enable-vc1dec --enable-v4l2
-  - make -j8 ; sudo make install
+  #make sure we did not break debug build
+  - ./autogen.sh --prefix=/opt/yami --enable-debug --enable-h264enc --enable-jpegenc --enable-vp8enc --enable-vp9enc --enable-h265enc --enable-mpeg2dec --enable-vp8dec --enable-vp9dec --enable-jpegdec --enable-h264dec --enable-h265dec --enable-vc1dec --enable-v4l2 && make -j8
```

Comment:
Can you add `--enable-tests` (maybe in both cases), too, since I've seen this break several times before.